recipes/openslide/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+import os
+
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
+
+required_conan_version = ">=1.53.0"
+
+
+class OpenSlideConan(ConanFile):
+    name = "openslide"
+    description = "OpenSlide is a C library for reading whole slide image files (also known as virtual slides)"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://openslide.org/"
+    topics = ("pathology", "whole-slide-imaging", "slide-image")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("cairo/1.18.0")
+        self.requires("gdk-pixbuf/2.42.10")
+        self.requires("glib/2.78.1", force=True)  # FIXME
+        self.requires("libdicom/1.0.5")
+        self.requires("libjpeg/9e")
```

Comment:
libjpeg-turbo is faster and widely used.  I'd recommend using it instead.

Reply:
Would love to, but due to dependency conflicts CCI has to stick to one variant of the library and has settled on `libjpeg` as the default. I personally hope that this is something that could be changed after the Conan 2.0 migration effort is complete.

I added an option for the choice of `libjpeg` variant at least.